examples/interactive.jl

```diff
@@ -746,6 +746,50 @@ end
         # Do not execute beyond this point!
         RecordEvents(scene, @replace_with_a_path)
     end
+
+    @cell "Ambient Occlusion" [SSAO, shading, interaction] begin
+        using AbstractPlotting
+
+        # SSAO (Screen Space Ambient Occlusion) has a couple of per-scene
+        # attributes.
+        # - `radius` sets the range of SSAO. You may want to scale this up or
+        #   down depending on the limits of your coordinate system
+        # - `bias` sets the minimum difference in depth required for a pixel to
+        #   be occluded. Increasing this will typically make the occlusion
+        #   effect stronger.
+        # - `blur` sets the range of the blur applied to the occlusion texture.
+        #   The texture contains a (random) pattern, which is washed out by
+        #   blurring. Small `blur` will be faster, sharper and more patterend.
+        #   Large `blur` will be slower and smoother. Typically `blur = 2` is
```

Comment:
Does blur need to always be Int32?

Reply:
Yes. The blurring algirithm has a loop `for pos in center-blur:center+blur`, where `pos` is a pixel position. 

Though since I've added defaults in AbstractPlotting it should convert to Int32, right?

Reply:
No, it will allow any type, so we should note that somewhere